configs/rollup.config.cjs.js

```diff
@@ -10,7 +10,7 @@ const plugins = [
 ]
 
 export default {
-	input: 'src/tram-one/index.js',
+	input: 'src/tram-one',
```

Comment:
we're no longer using `index.js` because we no longer have tests in the folders, so most folders were just the source and an index file.  It made functions unnecessarily nested, and it's cleaner now without them (err, at least flatter)